### PR TITLE
Settings - Stats addition (from versionInfo)

### PR DESCRIPTION
**Pull request by pusztaienike**

### CODE REVIEW

File: apps/sensenet/src/components/settings/stats-components-widget.tsx [hunk 1]
```diff
@@ -30,27 +51,74 @@ export interface ComponentsWidgetProps {
...
-        <div className={classes.rowContainer}>{localization.components}</div>
+        <div className={globalClasses.centeredVertical} style={{ padding: '10px 0' }}>
+          {localization.components}
+          <Tooltip title={localization.componentsInfo} placement="top">
+            <Info className={classes.info} />
+          </Tooltip>
+        </div>
+        <div className={classes.paragraph}>
+          {localization.latestBackendRelease}{' '}
+          {props.data.LatestReleases.find((item) => item.ProductName === 'SenseNet.Services') &&
```
Comment (taki9) [inline apps/sensenet/src/components/settings/stats-components-widget.tsx:70]: store it in a variable

File: apps/sensenet/src/components/settings/stats-components-widget.tsx [hunk 2]
```diff
@@ -30,27 +51,74 @@ export interface ComponentsWidgetProps {
...
+              props.data.LatestReleases.find((item) => item.ProductName === 'SenseNet.Services')!.ReleaseData,
+              'yyyy.MM.dd HH:mm aaa',
+            )}
+          <Link className={classes.link} href="https://www.sensenet.com/backend-updates" target="_blank" rel="noopener">
+            {localization.goToChangeLog}
+          </Link>
+        </div>
+        <div className={classes.paragraph}>
+          {localization.latestFrontendRelease}{' '}
+          {props.data.LatestReleases.find((item) => item.ProductName === 'SenseNet.AdminUI') &&
```
Comment (taki9) [inline apps/sensenet/src/components/settings/stats-components-widget.tsx:81]: store it in a variable

File: apps/sensenet/src/components/settings/stats-components-widget.tsx [hunk 3]
```diff
@@ -30,27 +51,74 @@ export interface ComponentsWidgetProps {
...
+          {localization.components}
+          <Tooltip title={localization.componentsInfo} placement="top">
+            <Info className={classes.info} />
+          </Tooltip>
+        </div>
+        <div className={classes.paragraph}>
+          {localization.latestBackendRelease}{' '}
+          {props.data.LatestReleases.find((item) => item.ProductName === 'SenseNet.Services') &&
+            dateUtils.formatDate(
+              props.data.LatestReleases.find((item) => item.ProductName === 'SenseNet.Services')!.ReleaseData,
```
Comment (taki9) [inline apps/sensenet/src/components/settings/stats-components-widget.tsx:72]: ReleaseData? backend typo? 😄

File: apps/sensenet/src/localization/default.ts [hunk 4]
```diff
@@ -453,6 +454,13 @@ const values = {
     clientSecret: 'Client Secret',
     generate: 'Generate access token',
     regenerate: 'Regenerate',
+    latestBackendRelease: 'Latest backend release:',
+    latestFrontendRelease: 'Latest AdminUI release:',
+    goToChangeLog: 'Go to changelog',
+    componentsInfo:
+      'Components of your sensenet. A component can consist of several packages, but there is also possible that a component is from only a single package.',
```
Comment (herflis) [inline apps/sensenet/src/localization/default.ts:461]:
```suggestion
      'Components of your sensenet instance. These components may include one or more packages.',
```

File: apps/sensenet/src/localization/default.ts [hunk 5]
```diff
@@ -453,6 +454,13 @@ const values = {
...
+    installedPackagesInfo:
+      'These packages make up the components of sensenet. There can also be packages that are not involved in the structure of the components, such as a tool-type package designed to run multiple times, for example deleting content or indexing.',
```
Comment (herflis) [inline apps/sensenet/src/localization/default.ts:463]:
```suggestion
      'These packages are mainly the building bricks of sensenet components. There are tool-like packages that are not part of the component structure, they were made to run multiple times, for example delete or index content.',
```

**Review by herflis — CHANGES_REQUESTED** [review]

File: apps/sensenet/src/components/settings/stats-components-widget.tsx [hunk 6]
```diff
@@ -21,6 +26,22 @@ const useStyles = makeStyles(() => {
       justifyContent: 'space-between',
       padding: '10px 0',
```
Comment (herflis) [inline apps/sensenet/src/components/settings/stats-components-widget.tsx:27]:
```suggestion
      padding: '10px 0',
      fontSize: '16px',
```

**Review by herflis — CHANGES_REQUESTED** [review]
Till we rethink the headings I think we could add the same font size to the `rowContainer` as it has in the API and Security page. This change should be made in
- stats-components-widget.tsx
- stats-installed-packages-widget.tsx
- stats-storage-widget.tsx

**Review by taki9 — APPROVED** [review]

**Review by herflis — APPROVED** [review]

**Pull request merged**